tests/test_clip_iqa.py

```diff
@@ -0,0 +1,59 @@
+import torch
+import pytest
+
+import numpy as np
+
+from PIL import Image
+from piq import CLIPIQA
```

Comment:
@denproc I added a test that checks that the file gets downloaded if not cached locally, which is what was not tested before. Regarding the CLIP feature extractor architecture, I lean towards not thoroughly testing it. There are two reasons for that:
1. We have a very limited usage of this architecture. For example, our metric never uses positional encoding by design. We could remove it but it is error-prone so I decided to leave it as is. All necessary blocks are tested just my running the model in existing tests.
2. The model architecture is taken from the official CLIP implementation with barely any changes. Since it is thoroughly tested by practice, I think we can be fairly confident that it works as expected.